src/controllers/login.js

```diff
@@ -14,7 +14,7 @@ exports.post = async (req, res) => {
       if (result) {
         req.session.username = loginusername;
         req.session.loggedIn = true;
-        res.redirect('/thankYou');
+        res.redirect('/usercontent');
```

Comment:
Let's discuss on gitter. I think we need to redirect to a page that explains that the 'gift' comes once a day, as per: 
<img width="298" alt="screen shot 2018-05-16 at 20 04 19" src="https://user-images.githubusercontent.com/19745717/40138396-73bd7e0a-5944-11e8-8a48-89790e0e353b.png">